packages/devtools_app/test/network/network_request_inspector_test.dart

```diff
@@ -76,7 +118,7 @@ void main() {
       expect(_clipboardContents, isEmpty);
       await tester.tap(find.byType(CopyToClipboardControl));
       final expectedResponseBody =
-          jsonDecode(utf8.decode(httpRequest!.responseBody!.toList()));
+      jsonDecode(utf8.decode(httpRequest!.responseBody!.toList()));
```

Comment:
FYI, `dart format` might complain about this if you haven't already formatted all your changes.

Reply:
Sorry I didn't pay attention. I have reformatted the files.